corehq/apps/reports/static/reports/v2/js/columns.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * todo add docstring
+ */
+
+hqDefine('reports/v2/js/columns', [
+  'lodash',  // can we switch to lodash?
+  'reports/v2/js/formatters',
+], function (
+    _,
+    Formatters
+) {
+  'use strict';
+
+  var Column = function (data) {
+    var self = this;
```

Comment:
FYI new best practice is to initialize to `{}` - `this` here is likely `window`

Reply:
fixed here https://github.com/dimagi/commcare-hq/pull/22063/commits/4769038b326df961c431997f51a4ad2cefff5730